sap/rest/gcts/simple.py

```diff
@@ -30,9 +31,25 @@ def fetch_repos(connection):
     return [Repository(connection, repo['name'], data=repo) for repo in result]
 
 
+def _wait_for_clone(repo, wait_for_ready, http_exc):
+    """Wait for clone process to finish"""
+
+    start_time = time.time()
+    while time.time() - start_time < wait_for_ready:
```

Comment:
We must let users know how long we are alrady waiting somehow. Check out the heartbeat functionality in sap/cli/gcts.py